### PR TITLE
feat(hub): create personalised section

### PR DESCRIPTION
**Pull request by vks-ovh**
ref: MANAGER-10184
 Addition of personalised section in the manager with services impacted by repricing

Signed-off-by: vikash singh <vikash.singh@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-10184
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

### 🎏  Translations

- [ ] TRANS-47786

### CODE REVIEW

File: packages/manager/apps/hub/src/dashboard/new-pricing/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerHubPricePageLazyLoading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('app.dashboard.newPricing.**', {
+      url: 'new-prices',
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/new-pricing/index.js:10]:
```suggestion
      url: '/new-prices',
```

File: packages/manager/apps/hub/src/dashboard/new-pricing/new-pricing.component.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import template from './template.html';
+
+export default {
+  bindings: {
+    getRepricing: '<',
+  },
+  controller() {},
+  controllerAs: '$ctrl',
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/new-pricing/new-pricing.component.js:8]: **nitpick:** If controller is not required, please remove it.

File: packages/manager/apps/hub/src/dashboard/new-pricing/routing.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dashboard.newPricing', {
+    url: 'new-prices',
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/new-pricing/routing.js:3]:
```suggestion
    url: '/new-prices',
```

File: packages/manager/apps/hub/src/dashboard/new-pricing/template.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    </p>
+    <p data-translate="manager_hub_new_pricing_explain"></p>
+    <ul>
+        <li data-translate="manager_hub_new_pricing_explain1"></li>
+        <li data-translate="manager_hub_new_pricing_explain2"></li>
+    </ul>
+</div>
+
+<oui-datagrid data-rows="$ctrl.getRepricing">
+    <oui-datagrid-topbar> </oui-datagrid-topbar>
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/new-pricing/template.html:16]:
```suggestion
```

File: packages/manager/apps/hub/src/routing.js [hunk 5]
```diff
@@ -69,6 +69,8 @@ export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
             serviceType: 'aapi',
           })
           .then(({ data }) => parseErrors(data)),
+      getRepricing: /* @ngInject */ ($http) =>
+        $http.get('/me/repricing/fall22').then(({ data }) => data),
```
Comment (sachinrameshn) [inline packages/manager/apps/hub/src/routing.js:73]: Question: Is it a valid URL ?

Comment (vks-ovh) [reply packages/manager/apps/hub/src/routing.js:73]: @sachinrameshn  yes, its a valid url

**Review by sachinrameshn — APPROVED** [review]

File: packages/manager/apps/hub/src/dashboard/new-pricing/index.js (hunk 1 above)
Comment (vks-ovh) [reply packages/manager/apps/hub/src/dashboard/new-pricing/index.js:10]: I tried adding `/new-prices` and it was not loading the page. so undo the changes.

**Review by sachinrameshn — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by sachinrameshn — DISMISSED** [review]

File: packages/manager/apps/hub/src/dashboard/new-pricing/template.html [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+        <span data-translate="manager_hub_new_pricing_description2"></span>
+    </p>
+    <p data-translate="manager_hub_new_pricing_explain"></p>
+    <ul>
+        <li data-translate="manager_hub_new_pricing_explain1"></li>
+        <li data-translate="manager_hub_new_pricing_explain2"></li>
+    </ul>
+</div>
+
+<oui-datagrid data-rows="$ctrl.getRepricing">
```
Comment (anooparveti) [inline packages/manager/apps/hub/src/dashboard/new-pricing/template.html:15]: **nitpick:** `getRepricing` sounds like a function. Can you please rename the variable?

Comment (vks-ovh) [reply packages/manager/apps/hub/src/dashboard/new-pricing/template.html]: @anooparveti  updated. Thanks

**Comment by vks-ovh** [comment]
This PR is not required anymore, hence closing it.

**Pull request closed without merging**